tests/integration/gemm/fp16/common.hpp

```diff
@@ -46,16 +46,17 @@ class TestBase {
     return name;
   }
   static constexpr mma_engine engine = mma_engine::xmx;
+  static constexpr gpu_arch gpu_arch = gpu_arch::XeHpg;
 };
 
 class Test0 : public TestBase {
  public:
   static constexpr size_t mat_m = 256;
   static constexpr size_t mat_n = 256;
   static constexpr size_t mat_k = 256;
-  static constexpr size_t wg_m = 256;
+  static constexpr size_t wg_m = 1;
   static constexpr size_t wg_n = 256;
-  static constexpr size_t sg_m = 32;
+  static constexpr size_t sg_m = 1;
```

Comment:
Revert changes on UT for testing. If we do need this case, then we should add it rather than change from an existing one.